### ISSUE TITLE
debian: change severity mapping

### ISSUE THREAD

**Issue by hdonnay** [quay/claircore#1069]
Really seems like the severity mapping should be:

```suggestion
| Debian Severity | Claircore Severity |
| -               | -                  |
| unimportant     | Negligible         |
| low             | Low                |
| medium          | Medium             |
| high            | High               |
| *               | Unknown            |
```

_Originally posted by @hdonnay in https://github.com/quay/claircore/pull/1067#discussion_r1334607884_

**Comment by hdonnay** [comment]
@RTann, do you recall what the thinking was on the current mapping?

**Comment by crozzy** [comment]
https://github.com/quay/claircore/discussions/828 context here

**Comment by RTann** [comment]
Yeah #828 covers the reasoning. I aligned Debian's definitions with Red Hat's and that's how the severity mapping was done. I think it makes sense, as users who may want to filter by critical vulns will never see Critical Debian's vulns if we never use it
